backend/dev/reset_db.py

```
@@ -0,0 +1,52 @@
+import os
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, text
```

Ruff (0.8.2)

`sqlalchemy.text` imported but unused

Remove unused import: `sqlalchemy.text`

(F401)